internal/db/query.go

```diff
@@ -227,17 +228,19 @@ func GetObjects(ctx context.Context, tx pgx.Tx, packManager *PackManager, projec
 			Deleted: deleted,
 			Content: content,
 		})
-	}, nil
+	}, closeFunc, nil
```

Comment:
My only concern is that there's no hard requirement for callers to actually call this function. Is there any way to avoid programmer error? I'm reminded of something like RAII in C++. Or will not calling a returned function make some lint rule light up? I haven't done much Go lately haha